### PR TITLE
Nano: Add output of cells of tutorial

### PR DESCRIPTION
**Pull request by y199387**
## Description
Add output of cells of Nano PyTorch train tutorial

## How was this PR tested?
Nano unit tests for notebooks

### CODE REVIEW

File: python/nano/notebooks/pytorch/tutorial/pytorch_train.ipynb [hunk 1]
```diff
@@ -13,12 +13,7 @@
    "metadata": {},
    "source": [
     "### Prepare Environment\n",
-    "Before you start with Apis delivered by bigdl-nano, you have to make sure BigDL-Nano is correctly installed for PyTorch. If not, please follow [this](../../../../../docs/readthedocs/source/doc/Nano/Overview/nano.md) to set up your environment.<br>\n",
-    "\n",
-    "We used pre-built cifar10 datamodule from lightning-bolts for demo. You are required to install lightnig-bolts as follows:\n",
-    "```python\n",
-    "pip install lightning-bolts\n",
-    "```"
+    "Before you start with Apis delivered by bigdl-nano, you have to make sure BigDL-Nano is correctly installed for PyTorch. If not, please follow [this](../../../../../docs/readthedocs/source/doc/Nano/Overview/nano.md) to set up your environment.<br>"
```
Comment (yangw1234) [inline python/nano/notebooks/pytorch/tutorial/pytorch_train.ipynb:16]: "Apis" -> "APIs"

File: python/nano/notebooks/pytorch/tutorial/pytorch_train.ipynb [hunk 2]
```diff
@@ -155,23 +190,91 @@
    "metadata": {},
    "source": [
     "### Train with Nano Apis\n",
```
Comment (yangw1234) [inline python/nano/notebooks/pytorch/tutorial/pytorch_train.ipynb:192]: "APIs"

File: python/nano/notebooks/pytorch/tutorial/pytorch_train.ipynb [hunk 3]
```diff
@@ -155,23 +190,91 @@
...
-    "The PyTorch Trainer (`bigdl.nano.pytorch.Trainer`) is the place where we integrate most optimizations. It extends PyTorch Lightning's Trainer and has a few more parameters and methods specific to BigDL-Nano. The Trainer can be directly used to train a `LightningModule`."
+    "The PyTorch Trainer (`bigdl.nano.pytorch.Trainer`) is the place where we integrate most optimizations. It extends PyTorch Lightning's Trainer and has a few more parameters and methods specific to BigDL-Nano. The Trainer can be directly used to train a `LightningModule`.\n",
+    "\n",
+    "`torch.channels_last` is recommended to be applied to the model object to raise CPU resource usage efficiency."
```
Comment (yangw1234) [inline python/nano/notebooks/pytorch/tutorial/pytorch_train.ipynb:195]: Does data_loader needs to be changed to use "channels_last"?

File: python/nano/src/bigdl/nano/pytorch/utils.py [hunk 4]
```diff
@@ -58,15 +60,21 @@ class ChannelsLastCallback(pl.Callback):
     def setup(self, trainer, pl_module, stage: Optional[str] = None) -> None:
         """Override hook setup to convert model to channels_last and wrap DataHook."""
         # TODO: Add check for module_states
+        try:
```
Comment (y199387) [inline python/nano/src/bigdl/nano/pytorch/utils.py:63]: The weights_prepack optimization of ipex will change the rank of convolution's weights to 5, e.g., [64, 3, 3, 3] to [1, 3, 3, 3, 64],  which make `model.to(...)` fail.

**Comment by yangw1234** [comment]
LGTM

**Comment by y199387** [comment]
Jenkins
http://10.112.231.51:18888/job/BigDL-NanoTests-PR-Validation/327/

**Pull request merged**